[2025-04-29T00:03:10.711+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2025-04-28T00:00:00+00:00 [queued]>
[2025-04-29T00:03:10.964+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2025-04-28T00:00:00+00:00 [queued]>
[2025-04-29T00:03:10.980+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2025-04-29T00:03:10.981+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2025-04-29T00:03:10.981+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2025-04-29T00:03:11.408+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2025-04-28 00:00:00+00:00
[2025-04-29T00:03:11.774+0000] {standard_task_runner.py:55} INFO - Started process 4329 to run task
[2025-04-29T00:03:11.826+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2025-04-28T00:00:00+00:00', '--job-id', '273', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpgmyv6a1e']
[2025-04-29T00:03:11.887+0000] {standard_task_runner.py:83} INFO - Job 273: Subtask create_table
[2025-04-29T00:03:14.084+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2025-04-28T00:00:00+00:00 [running]> on host fae2b1eff5d6
[2025-04-29T00:03:16.872+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2025-04-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-04-28T00:00:00+00:00
[2025-04-29T00:03:17.024+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2025-04-29T00:03:17.068+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstName TEXT NOT NULL,
                lastName TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2025-04-29T00:03:17.099+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2025-04-29T00:03:17.254+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20250428T000000, start_date=20250429T000310, end_date=20250429T000317
[2025-04-29T00:03:17.590+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2025-04-29T00:03:18.850+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
